charts/exivity/values.yaml

```diff
@@ -1,64 +1,138 @@
+nameOverride:     ""
+fullnameOverride: ""
+
+# Please overwrite this, and use your actual license, as it can only be changed through the Helm chart
+license:          ""
+
 ingress:
-  enable: true
-  host: exivity
-  ingressClassName: nginx
-  tlsSecret: exivity-tls
-  generateCert: true
-  serviceType: LoadBalancer
+  enabled:          true
+  host:             exivity
+  ingressClassName: ""
+
+  annotations: { }
+  #  kubernetes.io/ingress.allow-http:         'false'
+  #  nginx.ingress.kubernetes.io/ssl-redirect: 'true'
+  #  nginx.ingress.kubernetes.io/use-regex:    'true'
+
+  tls:
+    enabled: false
+    # Existing TLS secret name, or set to '-' to generate a cert
+    secret:  '-'
```

Comment:
double quotes instead of simple ones?

Reply:
Well, it's a single character. And Helm/Kubernetes don't care.